Documentation/EyeTracking/EyeTracking_EyeGazeProvider.md

```diff
@@ -1,11 +1,11 @@
 # Accessing eye tracking data in your Unity script
 
 The following assumes that you followed the steps for setting up eye tracking in your MRTK scene (see [Basic MRTK setup to use eye tracking](EyeTracking_BasicSetup.md)).
-To access eye tracking data in your MonoBehaviour scripts is easy! Simply use *MixedRealityToolkit.InputSystem.EyeGazeProvider*.
+To access eye tracking data in your MonoBehaviour scripts is easy! Simply use *CoreSystems.InputSystem.EyeGazeProvider*.
 
-## MixedRealityToolkit.InputSystem.EyeGazeProvider
+## CoreSystems.InputSystem.EyeGazeProvider
 
-While the *MixedRealityToolkit.InputSystem.EyeGazeProvider* provides several helpful variables, the key ones for eye tracking input are the following:
+While the *CoreSystems.InputSystem.EyeGazeProvider* provides several helpful variables, the key ones for eye tracking input are the following:
```

Comment:
I think it is CoreServices and not CoreSystems for the code examples.

Reply:
thanks, i'll confirm and fix

Reply:
fixed